Makefile

```diff
@@ -23,6 +25,7 @@ check-format:
 			$(foreach file,$(unformatted),$(\n)    gofmt -w $(file))$(\n)),\
 		@echo All files are well formatted.\
 	)
+
 .PHONY: test
 test:
-	go test -coverprofile=coverage.txt -covermode=atomic -race ./...
+	go test $(TEST_FLAGS) -coverprofile=coverage.txt -covermode=atomic -race ./...
```

Comment:
I'd like to have an explicit Makefile goal for the integration tests, so once we provision a cluster we also can issue something like `make integration` instead of `make test TEST_FLAGS="-tags integration"` :)

What do you think?

Reply:
Sounds good to me